### PR TITLE
MES-5336 Office Page Non-CPC

### PR DESCRIPTION
**Pull request by tomcrawleyy**
## Description
Introduces various components from the Debrief/Pass finalisation page to the Del Ex Office page for non-cpc journeys.
**NOTE Independent driving to be removed in later story, serious and dangerous comments to be amended in later story & health declaration toggle to be added in later story**
## Checklist

- [x] PR title includes the JIRA ticket number
- [x] Branch is rebased against the latest develop
- [x] Code has been tested manually
- [x] PR link added to JIRA ticket
- [ ] One review from each scrum team
- [x] Squashed commit contains the JIRA ticket number

## Screenshots (optional)
Cat B+E Pass
image
image

Cat B+E Fail
image
image

Cat B+E Terminate
image
image

Cat B+E Terminate Validation (works same way across cats)
image

### CODE REVIEW

**Comment by andrewsetterfield** [comment]
Can you change "Test Language" to "Language".  Also I've asked luke if this is needed if the conducted language is not welsh.

**Comment by andrewsetterfield** [comment]
Can you add the Application reference number to the top of the page.

File: src/pages/office/cat-be/office.cat-be.page.html [hunk 1]
```diff
@@ -26,42 +26,96 @@ <h4>Test summary</h4>
...
+              [display]=true
+              [d255]="pageState.d255$ | async"
+              [outcome]="testOutcome"
+              [formGroup]="form"
+              (d255Change)="d255Changed($event)"></d255>
+
+            <language-preferences
+              *ngIf="isDelegated"
+              [formGroup]="form"
+              [isWelsh]="(pageState.conductedLanguage$ | async) === 'Cymraeg'"
```
Comment (andrewsetterfield) [inline src/pages/office/cat-be/office.cat-be.page.html]: can you use the enum for this instead of hardkeying the value

**Comment by andrewsetterfield** [comment]
> Can you change "Test Language" to "Language". Also I've asked luke if this is needed if the conducted language is not welsh.

My bad, its Test language on the pass finilisation screen so am getting luke to confirm as they prob shouldnt be inconsistant.

**Comment by matthew2564** [comment]
Should the pass certificate number field be appearing on terminated tests?

**Review by andrewsetterfield — APPROVED** [review]

**Pull request closed without merging**